packages/server/src/middlewares/userInfo.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response, NextFunction } from 'express';
```

Comment:
use the below syntax instead
`import type { Request, Response, NextFunction } from 'express';`